metta/DeductionEntail.metta

```diff
@@ -24,34 +24,29 @@
 ;;;;;;;;;;;;;;;;;;;;;
 
 (⊢ ;; premises
-   (≞ $P $ptv)
-   (≞ $Q $qtv)
-   (≞ $R $rtv)
+   (≞ ($P $a) $ptv)
+   (≞ ($Q $a) $qtv)
+   (≞ ($R $a) $rtv)
```

Comment:
@tanksha, it should be `$P` instead of `($P $a)`, etc, as this rule cares about the predicates themselves, not instances of these predicates.  Only direct inference rules take into account instances.